src/extensions/query-translator/component/LoadingIcon.tsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+
+export const GPT_LOADING_ICON = (
+  <div className='centered' style={{ textAlign: 'center' }}>
+    <br />
+    <img
+      style={{ width: 40, animation: 'pulse 2s infinite', marginTop: 'auto', marginLeft: 'auto', marginRight: 'auto' }}
+      src='https://seeklogo.com/images/O/open-ai-logo-8B9BFEDC26-seeklogo.com.png'
```

Comment:
probably we should store the logo locally inside `public/openai.png`.

Reply:
done